### ISSUE TITLE
Average processing time calculated incorrectly

### ISSUE THREAD

**Issue by jadamcrain** [nats-io/nats.rs#1151]
### Observed behavior

It always divides by 2, but it should divide by the number of requests served:

https://github.com/nats-io/nats.rs/blob/67b2798004c9543bd90260b15b3c2a6888ef07a5/async-nats/src/service/mod.rs#L787

I believe this should be:

```
stats.average_processing_time = stats.processing_time.checked_div(stats.requests).unwrap();
```

### Expected behavior

The avg time should total time / # requests

### Server and client version

current tip of `main`

**Comment by jadamcrain** [comment]
If the endpoint returns `total requests` and `total time` it seems like the CLI should calculate the average?

Then every language implementation wouldn't have to do it since it's a "derived quantity" for the underlying data.

**Comment by Jarema** [comment]
oh, that is a silly mistake.
Will issue a fix shortly, unless you want to contribute it 🙂

**Comment by jadamcrain** [comment]
I'll do it just to get a PR, why not.

**Comment by jadamcrain** [comment]
PR in https://github.com/nats-io/nats.rs/pull/1152

**Comment by Jarema** [comment]
Fixed in #1152 

Thank you @jadamcrain !
